### DISCUSSION TITLE
There is a webpage for showing community plugins?

### DISCUSSION THREAD

**Discussion by palcarazm** [twbs/bootstrap#37675] [Q&A] [answered]
It will be amazing having a bootstrap webpage where to see and show community plugins. You will be able to find the plugin you need for you bootstrap webapp! 

What do you think? It exists something similar?

**Comment by mdo** [comment, marked as answer]
Doesn't exist currently. Want to start compiling a list? We could get a page together.

**Reply by palcarazm** [reply]
Hi,

Yes we could start a page together! 🥳

Are you thinking in a webapp or just a simple list?

**Reply by lonix1** [reply]
To keep it simple, it might be good enough to have a simple list. Think about all those "awesome" lists... maybe something like that?

**Reply by lonix1** [reply]
There's some resources here, but out of date: https://github.com/therebelrobot/awesome-bootstrap

**Reply by palcarazm** [reply]
> There's some resources here, but out of date: https://github.com/therebelrobot/awesome-bootstrap

Thanks! I will take a look

**Reply by filipkappa** [reply]
Feel free to issue a PR for the awesome-bootstrap list :) it will be updated & maintained more regularly starting in February 2023
https://github.com/therebelrobot/awesome-bootstrap/

**Reply by palcarazm** [reply]
What an awesome list of resource! I've start drafting a static webpage that take resource from npm registry. I expected i can show you more asap.

**Reply by filipkappa** [reply]
Great, I've reached out to you via the mail listed on your GitHub account so that we can stop spamming the Bootstrap devs ;)

**Reply by palcarazm** [reply]
Hi @filipkappa,

I've tried to contact you via mail with no results.  In https://palcarazm.github.io/awesome-bootstrap/ you can find a little demo of the solution I asked you for the bootstrap plugin webpage.

What do you think about it?

**Reply by filipkappa** [reply]
Apologies, I missed your message & now I responded to it via email ;)
